### DISCUSSION TITLE
Develop Environment?

### DISCUSSION THREAD

**Discussion by Kent-Liu** [openvanilla/McBopomofo#235] [General]
Dears, 

I think I maybe able to provide some help.
But the code of master branch  today, is not able to compiled under Xcode 12.5.1 or Xcode 13.1.
Would you please help to advise how to prepare suitable environment for develop?
Thanks.

**Comment by lukhnos** [comment]
If you saw messages like `Missing Package Product 'x'`, try Clean Build Folder in Xcode. If it doesn't work, try `git clean -fdx .` your git directory, which will clean up all non-git-managed files—which was what I had to use.

If it still doesn't work, you may want to post what you saw.

**Reply by Kent-Liu** [reply]
@lukhnos 
Thanks for your advisement. Just 'Clean Build' under Xcode 12.5.1 or Xcode 13.1 are not work.
But after I upgrade to Xcode 13.2.1, 'Clean Build' solve the issue. Thanks a lot. :)
